base/src/main/java/com/smartdevicelink/managers/lifecycle/BaseSystemCapabilityManager.java

```diff
@@ -266,81 +272,90 @@ void parseRAIResponse(RegisterAppInterfaceResponse response) {
 		}
 	}
 
-	private void setupRpcListeners() {
```

Comment:
The alignment of `private void setupRpcListeners()` was altered .  It (and all of its code) needs to be on the same level as the function above it (`parseRAIResponse(...)`).

Reply:
> The alignment of `private void setupRpcListeners()` was altered . It (and all of its code) needs to be on the same level as the function above it (`parseRAIResponse(...)`).

@kostyaBoss I still see this issue.  The alignment of private void `setupRpcListeners()` is still a little bit off. It (and all of its code) needs to be on the same level as the function above it (`parseRAIResponse(...)`).